src/main/java/com/flightstats/hub/config/properties/MetricsProperties.java

```diff
@@ -0,0 +1,45 @@
+package com.flightstats.hub.config.properties;
+
+import com.flightstats.hub.app.HubHost;
+
+import javax.inject.Inject;
+
+public class MetricsProperties {
+
+    private final PropertiesLoader propertiesLoader;
+    private final AppProperties appProperties;
+
+    @Inject
+    public MetricsProperties(PropertiesLoader propertiesLoader, AppProperties appProperties) {
+        this.propertiesLoader = propertiesLoader;
+        this.appProperties = appProperties;
+    }
+
+    public boolean isEnabled() {
+        return propertiesLoader.getProperty("metrics.enable", false);
+    }
+
+    public String getEnv(){
+        return appProperties.getEnv();
+    }
+
+    public int getReportingIntervalInSeconds() {
+        return propertiesLoader.getProperty("metrics.seconds", 15);
+    }
+
+    public String getRoleTag() {
+        return propertiesLoader.getProperty("metrics.tags.role", "hub");
+    }
+
+    public String getTeamTag() {
+        return propertiesLoader.getProperty("metrics.tags.team", "development");
+    }
+
+    public String getHostTag() {
+        return HubHost.getLocalName();
```

Comment:
Can you inject an instance of HubHost? Even if that instance is just HubHost.getInstance() provided by a ```@Provider``` it'd start the path of decoupling from the static methods. Though you'd have to define non-static verisons of methods like ```getLocalName()```.

Reply:
yes i thought of that. Created a task for it.